src/main/java/org/opensearch/security/ssl/DefaultSecurityKeyStore.java

```diff
@@ -907,6 +911,28 @@ private SslContext buildSSLServerContext(final File _key, final File _cert, fina
         return buildSSLContext0(_sslContextBuilder);
     }
 
+    private SslContextBuilder configureSSLServerContextBuilder(final SslContextBuilder builder, final SslProvider sslProvider,
+            final Iterable<String> ciphers, final ClientAuth authMode) {
+        return builder
+            .ciphers(Stream
+                .concat(
+                    Http2SecurityUtil.CIPHERS.stream(),
+                    StreamSupport.stream(ciphers.spliterator(), false))
+                .collect(Collectors.toSet()), SupportedCipherSuiteFilter.INSTANCE)
+            .clientAuth(Objects.requireNonNull(authMode)) // https://github.com/netty/netty/issues/4722
```

Comment:
Seems like the mentioned netty issue was fixed a while ago but this comment wasn't removed. Should we remove it? I know it maybe out of scope of this PR but wanted to know your thoughts